example/e2e/init.js

```diff
@@ -2,18 +2,18 @@ const detox = require('detox');
 const config = require('../../package.json').detox;
 const adapter = require('detox/runners/jest/adapter');
 const specReporter = require('detox/runners/jest/specReporter');
+const assignReporter = require('detox/runners/jest/assignReporter');
 
-// Set the default timeout
-jest.setTimeout(120000);
-jasmine.getEnv().addReporter(adapter);
+detoxCircus.getEnv().addEventsListener(adapter);
+detoxCircus.getEnv().addEventsListener(assignReporter);
+detoxCircus.getEnv().addEventsListener(specReporter);
 
-// This takes care of generating status logs on a per-spec basis. By default, jest only reports at file-level.
-// This is strictly optional.
-jasmine.getEnv().addReporter(specReporter);
+// Set the default timeout
+jest.setTimeout(130000);
 
 beforeAll(async () => {
   await detox.init(config);
-});
+}, 300000);
```

Comment:
This timeout overrides the `jest.setTimeout(130000)`. Why you need this if you set the previous timeout using `jest.setTimeout(130000)` ?

Reply:
this timeout applies only to the `beforeAll` step - which can take long because the emulator is starting up

130000 is used for the tests themselves

Reply:
Ah, ok!